Assets/MixedRealityToolkit-SDK/Features/Utilities/Solvers/Overlap.cs.meta

```diff
@@ -0,0 +1,11 @@
+fileFormatVersion: 2
+guid: 97026b56b4e804b468b8d53af13e3c3b
+MonoImporter:
+  externalObjects: {}
+  serializedVersion: 2
+  defaultReferences: []
+  executionOrder: 0
+  icon: {instanceID: 0}
```

Comment:
Be sure to set an Icon :)

Reply:
Why do we override the default script icon?